app/main.py

```diff
@@ -1 +1,57 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> None | int:
```

Comment:
I'm not sure about None here. Could you please explain why do you use it cause i can't understand

Reply:
You are right! 
I just forgot to delete it. 
The previously was supposed to  return None. So "None" is just legacy.
I will fix